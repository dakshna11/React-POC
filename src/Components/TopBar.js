import React from 'react';
function TopBar() {
    return (
        <div className="topbar">
            <div><span id="logo-1">Auto</span><span id="logo-2">Wash</span></div>

            <div>
                <div class="top-bar-icon">
                    <i class="far fa-clock"></i>
                </div>
                <h3>Opening Hour</h3>
                <p>Mon - Fri, 8.00 - 9.00</p>
            </div>
             
            <div><h3>Call Us</h3><p>+012 345 6789</p></div>
            <div><h3>Email Us</h3><p>info@example.com</p></div>
        </div>
        
    );
}

export default TopBar;