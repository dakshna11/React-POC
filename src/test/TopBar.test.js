import React from 'react';
import { shallow } from 'enzyme';
import TopBar from "../Components/TopBar";



let wrapped = shallow(<TopBar></TopBar>);
describe('TopBar', () => {
  it('should render the TopBar Component correctly', () => {   
    expect(wrapped).toMatchSnapshot();
  });
})