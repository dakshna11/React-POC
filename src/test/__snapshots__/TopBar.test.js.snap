// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TopBar should render the TopBar Component correctly 1`] = `
<div
  className="topbar"
>
  <div>
    <span
      id="logo-1"
    >
      Auto
    </span>
    <span
      id="logo-2"
    >
      Wash
    </span>
  </div>
  <div>
    <div
      class="top-bar-icon"
    />
    <h3>
      Opening Hour
    </h3>
    <p>
      Mon - Fri, 8.00 - 9.00
    </p>
  </div>
  <div>
    <div>
      <h3>
        Call Us
      </h3>
      <p>
        +012 345 6789
      </p>
    </div>
  </div>
  <div>
    <h3>
      Email Us
    </h3>
    <p>
      info@example.com
    </p>
  </div>
</div>
`;
